test/metabase/query_processor_test/advanced_math_test.clj

```diff
@@ -1,7 +1,9 @@
 (ns metabase.query-processor-test.advanced-math-test
-  (:require [clojure.test :refer :all]
-            [metabase.test :as mt]
-            [metabase.util :as u]))
+  (:require
+   [clojure.test :refer :all]
+   [metabase.driver :as driver]
+   [metabase.test :as mt]
+   [metabase.util :as u]))
```

Comment:
This is the new semi-official way to indent `:require` forms that the community has settled on. My editor automatically indents stuff like this now when I have it clean the namespace form. For a while I went in and manually changed the indentation to minimize the diff size. But I'm starting to think we should just bite the bullet and start moving things over to the new style regardless of diff concerns.